llvm/unittests/Option/Opts.td

```diff
@@ -50,6 +51,16 @@ def Blurmpq_eq : Flag<["--"], "blurmp=">;
 def Q : Flag<["-"], "Q">, Visibility<[SubtoolVis]>;
 def R : Flag<["-"], "R">, Visibility<[DefaultVis, SubtoolVis]>;
 
+def multiline_help : Flag<["-"], "multiline-help">, HelpText<"This a help text that has\nmultiple lines in it">, Visibility<[MultiLineVis]>;
+
+def multiline_help_long : 
```

Comment:
Delete trailing space on this line and the next line

Reply:
Fixed